src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -26,6 +26,20 @@ public boolean connect(String database, String user, String password) {
         }
     }
 
+    /**
+     * Method for inserting data into a table.
+     * If successful, the method returns true, else return false
+     */
+    public boolean insert(String tableName, DataSet dataset) {
+        try (Statement statement = connection.createStatement()) {
+            statement.execute(String.format("INSERT INTO %s VALUES(%s)", tableName, dataset.toString()));
```

Comment:
I'm not sure that using overrided method toString() as a parameter in that case is good idea. Method toString() should returns a string representation of the object... I mean that it's not good that logic of the method depends on toString(). Probably it's better to extract this logic to seprate method

Reply:
I understand what you mean, but I do not want to create another method. Let's see what other say.

Reply:
I don't like toString implementation either. Now it's okay but it will add you problems when you add HashMap to DataSet. I don't know how to implement toString in that case.

Reply:
Maybe i rename this method and all will be fine :)

Reply:
Of course, rename. use toString only for printing logic or something similar. 
Not for business logic or comparing or something

Reply:
I did it. :)